src/pages/projects/[index].tsx

```diff
@@ -1,197 +1,65 @@
-            },
-            {
-                label: 'https://github.com/Open-SGF/motherhood-reclaimed-website',
-                url: 'https://github.com/Open-SGF/motherhood-reclaimed-website',
-            },
-        ],
-        toolsUsed: [
-            {
-                label: 'CraftCMS',
-                image: {
-                    src: 'https://via.placeholder.com/70.png',
-                    alt: 'Example Alt Text',
-                },
-            },
-            {
-                label: 'Node',
-                image: {
-                    src: 'https://via.placeholder.com/70.png',
-                    alt: 'Example Alt Text',
-                },
-            },
-            {
-                label: 'MySQL',
-                image: {
-                    src: 'https://via.placeholder.com/70.png',
-                    alt: 'Example Alt Text',
-                },
-            },
-        ],
-        contributors: [
-            {
-                name: 'Ethan Zitting',
-                image: {
-                    src: 'https://via.placeholder.com/85.png',
-                    alt: 'Image alt text.',
-                },
-                linkUrl: 'https://github.com/ethanzitting',
-            },
-            {
-                name: 'Levi Zitting',
-                image: {
-                    src: 'https://via.placeholder.com/85.png',
-                    alt: 'Image alt text.',
-                },
-                linkUrl: 'https://github.com/ethanzitting',
-            },
-            {
-                name: 'Shannon Treadway',
-                image: {
-                    src: 'https://via.placeholder.com/85.png',
-                    alt: 'Image alt text.',
-                },
-                linkUrl: 'https://github.com/ethanzitting',
-            },
-            {
-                name: 'Aaron Merrick',
-                image: {
-                    src: 'https://via.placeholder.com/85.png',
-                    alt: 'Image alt text.',
-                },
-                linkUrl: 'https://github.com/ethanzitting',
-            },
-        ],
-        clientIssue: 'Client needed a website to help share their mission to' + ' potential partners',
-        ourSolution:
-            'We worked with them to come up with a design they liked and' +
-            ' then launched their website with a CMS so they could edit the' +
-            ' content as needed.',
-    };
-
-    const router = useRouter();
 
+export const getServerSideProps: GetServerSideProps = async ({ query: { index: url } }) => {
+    let contributors;
     let projectData;
 
-    if (router.asPath.includes('mo-jobs')) {
+    if (url === 'mo-jobs') {
+        const apiContributors = await getProjectContributors('portal-to-work-api');
+        const clientContributors = await getProjectContributors('portal-to-work-client');
+
+        contributors = mergeContributors(apiContributors, clientContributors);
```

Comment:
In the cases of projects with multiple repos, how about we have a list of repo URL's in the incoming data so then we know that regardless if we have a project with 1..N repos, we'll either merge or not all the contributors together. Ideally we setup the concept of github projects for this case, and I'm not sure if github api will automatically know for the project in total or still only count individual repositories. Either way, this function only accepts up to two, but what if there's 3?